sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -337,7 +337,7 @@ def __init__(
     """Initializes ``WriteToPubSub``.
 
     Args:
-      topic: Cloud Pub/Sub topic in the form "/topics/<project>/<topic>".
+      topic: Cloud Pub/Sub topic in the form "projects/<project>/topics/<topic>".
```

Comment:
Looks like this line is one character too long, which is causing the Lint test to fail. once you fix this, the bot will assign someone to review!